shared-components/statviz/components/visualizations/movedBoxes/BoxFlowSankey.tsx

```diff
@@ -129,13 +129,22 @@ export default function BoxFlowSankey({ width, height, data, boxesOrItems }: IBo
 
   const nodes = [
     outgoingNode,
-    ...movedBoxes.map((movedBox) => ({
-      id: movedBox.targetId,
-      name: movedBox.isNegative ? `${movedBox.name} removed` : movedBox.name,
-      nodeColor: movedBox.isNegative
-        ? "red"
-        : sample(["#9467bd", "#8c564b", "#e377c2", "#7f7f7f", "#bcbd22"]),
-    })),
+    ...movedBoxes.map((movedBox) => {
+      const getName = () => {
```

Comment:
I would rather define this utility function somewhere outside. This way it is created during each render.